kernel/arm_boards/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+authors = ["Nathan Royer <nathan.royer.pro@gmail.com>"]
+name = "arm_boards"
+description = "Per-board AArch64 configuration"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+memory_structs = { path = "../memory_structs" }
+cpu = { path = "../cpu" }
```

Comment:
see my other comment -- we should invert this dependency such that `cpu` depends on this crate instead of this crate depending on `cpu`.